src/components/Molecules/MenuList/__snapshots__/MenuList.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<MenuList /> Matches the Menu List snapshot 1`] = `
+<div
+  aria-labelledby={null}
```

Comment:
Should we provide non-null defaults for aria tags?

Reply:
This doesn't do this in the HTML. Since it's just a jest thing, let's maybe ignore for now?